custom-vu/src/main/kotlin/jces1209/vu/action/WorkAnIssue.kt

```diff
@@ -91,4 +100,14 @@ class WorkAnIssue(
             }
         }
     }
+
+    private fun attachScreenShot(issuePage: AbstractIssuePage) {
+        val attachScreenShot = issuePage.addAttachment()
+        attachScreenShot.makeScreenShot()
+        meter.measure(ATTACH_SCREENSHOT)
+        {
```

Comment:
formatting - to avoid unnecessary conflicts and deltas
```suggestion
        meter.measure(ATTACH_SCREENSHOT) {
```